application.js

```diff
@@ -8,47 +8,55 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+  var email = "dleuliette@gmail.com";  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
-
+  $('#email').val('maret.benoit@gmail.com');
 
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
-
+  $('#email-hint').html('<strong>This is my email now</strong>');
 
   // Ex 4. Add the .blue CSS class to the table header cells (th elements)
-
+  $('th').addClass('blue');
 
 
   // Ex 5. Count the number of table body rows there are (team count!)
-  var team_count = 0;  // TODO: replace 0, keep the team_count variable.
-
+  var team_count = $('tbody tr').length;  // TODO: replace 0, keep the team_count variable.
 
 
   // Ex 6. Say there is a 15th team added to the table.
   //       Add a row at the bottom, this new team should have zero points.
-
+  $('tbody').append('<tr><td>15</td><td>Mon équipe </td><td>0</td></tr>');
```

Comment:
>
> Why you didn't use the team_countvariable ?
>

I don't really know, I was coding as I felt it at this moment.
I didn't think about it.

Nice Job but don't forget to write great commi messages
> http://davidl.fr/gitclub.html


I won't, I'll improve my naming skills ;)



2016-03-29 16:56 GMT+02:00 David Leuliette <notifications@github.com>:

> In application.js
> <https://github.com/flexbox/jquery-dom-manipulation/pull/2#discussion_r57737426>
> :
>
> >
> >
> >    // Ex 6. Say there is a 15th team added to the table.
> >    //       Add a row at the bottom, this new team should have zero points.
> > -
> > +  $('tbody').append('<tr><td>15</td><td>Mon équipe </td><td>0</td></tr>');
>
> Why you didn't use the team_countvariable ?
>
> var row = $('<tr>').html('<td>' + (team_count + 1) + '</td><td>New team</td><td>0</td></tr>');
> $('tbody').append(row);
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/flexbox/jquery-dom-manipulation/pull/2/files/c42e3b3e675953e153377abd2dfac2d4db90d4cc#r57737426>
>